app/views/layouts/components/_container.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="govuk-grid-row">
```

Comment:
In our other rendered components it looks like we apply the row in `_component_renderer` - might be worth moving that into there so it's the same as the others

Reply:
Makes sense! Resolved in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/52/commits/1a1349ee5e6799285afe599e510b66047299b42d.